hangman.rb

```diff
@@ -0,0 +1,262 @@
+              "|     |      ",
+              "|     |",
+              "| ____|___"
+            ],
+                                    [
+              "|     _________",
+              "|     |/      |",
+              "|     |      (_)",
+              "|     |      "+ "\\".red + "|",
+              "|     |       |",
+              "|     |      ",
+              "|     |",
+              "| ____|___"
+            ],
+                                                [
+              "|     _________",
+              "|     |/      |",
+              "|     |      (_)",
+              "|     |      \\|" + "/".red,
+              "|     |       |",
+              "|     |      ",
+              "|     |",
+              "| ____|___"
+            ],
+                                                [
+              "|     _________",
+              "|     |/      |",
+              "|     |      (_)",
+              "|     |      \\|/ ",
+              "|     |       | ",
+              "|     |      " + "/ ".red,
+              "|     |",
+              "| ____|___"
+            ],
+                                                [
+              "|     _________",
+              "|     |/      |",
+              "|     |      (_)",
+              "|     |      \\|/",
+              "|     |       |",
+              "|     |      / " + "\\ ".red,
+              "|     |",
+              "| ____|___"
+]]
+
+    start
+  end
+
+  def pick_word
+    word_list = %w{cat dog egg horse donkey elephant pig monkey iguana cow goat unigoat eggplant mouse dragon phoenix}
+    random_word = word_list.sample
+    return random_word
+  end
+
+  def start
+    puts "Welcome to Hangman."
+    puts "Please type your letter guess. No numbers, punctuation, or spaces."
+    puts "Answer: #{@game_word}"
+
+    @now_playing = true
+    puts "To exit, type: quit."
+    draw_tree(0)
+
+    draw_slots
+    draw_wrong_guesses
+    puts " "
+    play
+  end
+
+  def play
+    while @now_playing
+      guess = get_guess
+      check_guess(guess)
+      update_board
+      check_win
+    end
+  end
+
+  # ---------------------------------
+  # Guess Methods
+
+  def get_guess
+    input = gets.chomp.strip.downcase
+    if input == "quit" || input == "exit"
+      exit
+    else
+      guess = validate_input(input)
+    end
+    return guess
+  end
+
+  def validate_input(input)
+    valid = false
+
+    while !valid
+      input.split("").each do |letter|
+        # enter and space do not count as mistakes
+        # correctly identifies 1st invalid input
+        # submits 2nd as a wrong guess
+        if letter == " " || !("a".."z").include?(letter)
+          puts "\nError:".red + " Please only use letters. No numbers, punctuation, or spaces."
+          input = gets.chomp.strip.downcase
+        end
+      end
+
+      valid = true
+      return input
+    end
+  end
+
+  def check_guess(guess_letter)
+    # is guess_letter one lettr or a word/phrase
```

Comment:
Since you have two distinct paths here: a word or a letter, it might make sense to separate those out into two additional `check_word_guess` and `check_letter_guess` or something like that.